books/models.py

```diff
@@ -20,11 +20,14 @@ class LoanStatus(ChoiceEnum):
 
 
 class BookQueryset(models.QuerySet):
+    def get_books_in_category(self, category):
+        return self.filter(category__name=category)
+
     def get_books_in_supercategory(self, supercategory):
         if supercategory == "Nonfiction":
             books = self.get_books_nonfiction()
```

Comment:
One for later maybe, but this function should probably also be retired in favour of query `__in=` parameters :)

Reply:
Ah okay!